app/app.js

```
@@ -303,6 +303,16 @@ bpmnModeler.on('import.parse.complete', (event) => {
 
 bpmnModeler.importXML(diagramXML).then(() => {});
 
+
+const btn = document.getElementById('messageButton');
+btn.addEventListener('click', (_event) => {
+  launchMarkdownEvent.eventBus.fire('spiff.add_message.returned', {
+    name: "C",
```

ESLint

[error] Replace `"C"` with `'C'`

(prettier/prettier)